modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -5,6 +5,7 @@ module DecisionReviews
     module NoticeOfDisagreements
       class EvidenceSubmissionsController < AppealsApi::ApplicationController
         skip_before_action :authenticate
+        before_action :set_notice_of_disagreement, only: :create, if: :nod_id_included?
```

Comment:
This pattern of deliberately nil'ing out the NOD if the param is missing and then relying on the raise afterwards seems a little brittle to me. Is there a better way to short circuit the #create if the `nod_id` is missing? I'm not sure we need the callback.